meeting/app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,42 @@
+class GuestsController < ApplicationController
+    def new
+        @host = Host.all
+    end
+    
+    def create
+        @host = Host.all
+        guest = Guest.new
+        guest.picture = params[:picture]
+        guest.name = params[:name]
+        guest.email = params[:email]
+        guest.host_id = params[:host_id]
+        guest.save
+        redirect_to '/guests'
+    end
+    
+    def index
+        @guests = Guest.all
+    end
+    
+    def edit
+        @host =Host.all
```

Comment:
For code clarity it' best to leave a space after the `=` sign

Reply:
Thanks.